assets/src/hooks/useNearestIntersection.ts

```diff
@@ -17,15 +17,17 @@ export const useNearestIntersection = (
       return { ...oldResult, is_loading: true }
     })
 
-    fetchNearestIntersection(latitude, longitude).then((result) => {
-      if (shouldUpdate) {
-        if (result) {
-          setResult({ ok: result })
-        } else {
-          setResult({ is_error: true })
+    if (latitude && longitude) {
+      fetchNearestIntersection(latitude, longitude).then((result) => {
+        if (shouldUpdate) {
+          if (result) {
+            setResult({ ok: result })
+          } else {
+            setResult({ is_error: true })
+          }
         }
-      }
-    })
+      })
+    } else setResult({ ok: null })
```

Comment:
Hmm... I see why `ok: null` is odd, but I also didn't consider this an error, because nothing has gone wrong at this point. We're just calling `useNearestIntersection` too soon, before startPoint has been set. What if I set back to initial state of `{is_loading: true}`?

Reply:
> What if I set back to initial state of `{is_loading: true}`?

👍🏻 I think this works for me! I'd personally prefer to use `loading()` from `fetchResult` instead of the written out object.

Reply:
I wish there was a 💡 emoji reaction on PRs, I'd use that all the time